app/_locales/en/messages.json

```diff
@@ -1839,6 +1842,17 @@
     "message": "Swap $1 to $2",
     "description": "Used in the transaction display list to describe a swap. $1 and $2 are the symbols of tokens in involved in a swap."
   },
+  "swapTokenVerificationMessage": {
+    "message": "$1 Always confirm the token address on $2. $3",
```

Comment:
$1 here seems like an entirely separate sentence. Could this be broken out of this message, to simplify the work for translators? I'm not entirely sure why it's embedded.

Reply:
Similarly, $2 and $3 can be represented as a single placeholder. There is no need for them to be separated.